frontend_tests/node_tests/emoji.js

```diff
@@ -0,0 +1,25 @@
+set_global('$', global.make_zjquery());
+set_global('page_params', {});
+set_global('upload_widget', {});
+
+add_dependencies({
+    emoji_codes: 'generated/emoji/emoji_codes.js',
+    emoji: 'js/emoji.js',
```

Comment:
You can put this after `add_dependencies` as a requirement, like `var emoji = require('js/emoji.js');`. This is what we usually do with the file we are testing.